util/grub-mkconfig_lib.in

```diff
@@ -293,10 +293,22 @@ version_test_gt ()
   return "$?"
 }
 
+version_delphix_latest ()
+{
+  zcat /usr/share/doc/delphix-entire-*/packages.list.gz | grep '^delphix-kernel-' | cut -d= -f1 | sed 's/delphix-kernel-//'
```

Comment:
I'm not necessarily opposed to this, but I think I'd rather not use the `delphix-entire-*` glob here, when we could more precisely specify the package we want, by first querying what platform we're running on..

we could even add a new `get-appliance-platform` script in the `delphix-platform` repository if that'd make it cleaner (as we've discussed a bit before).. since it's a little awkward otherwise, for grub to know about how we store that info (i.e. the path to the file) on the appliance.

I don't oppose landing as-is, though.

Reply:
I think `get-appliance-platform` is a good idea, I wanted to have a command like this for some time now.

Regarding the hotfix workflow, yeah I think we'd want to update `delphix-entire` when we install a hotfix.